content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,255 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+In machine learning projects, number of <abbr>experiments</abbr> grow rapidly.
+DVC can track these experiments, list their most relevant parameters and
+metrics, and commit only the ones that we need to Git.
+
+In this section, we explore the basic features of DVC experiment management with
+[`get-started-experiments`][gse] project.
+
+[gse]: https://github.com/iterative/get-started-experiments
+
+<details>
+
+## ⚙️ Installing the Example Project
+
+These commands are run in the [`get-started-experiments`][gse] project. You can
+run the commands in this document after cloning the repository and installing
+the requirements.
+
+### ⚡ Clone the project and create virtual environment
+
+Please clone the project and create a virtual environment.
+
+> We create a virtual environment to keep the libraries we use isolated from the
+> rest of your system. This prevents version conflicts.
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments -b get-started
+$ cd get-started-experiments
+$ virtualenv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
+```
+
+### ✅ Get the data set
+
+The repository we cloned doesn't contain the dataset. We use `dvc pull` to
+update the missing data files.
+
+```dvc
+$ dvc pull
+```
+
+The repository already contains the necessary configuration to run the
+experiments.
+
+## 💡 Preparing a project for DVC experiments
+
+If DVC is not installed to the system, please refer to [install](/doc/install).
+
+In this getting started page, we assume that there is already a configured DVC
+project to simplify the introduction. DVC experiments require a DVC pipeline
+defined in the project.
+
+If DVC is not initialized before in the project, you can do so by:
+
+```dvc
+$ dvc init
+```
+
+DVC also requires commands to be run and their dependencies to be defined as
+stages. We use `dvc stage add` to add a stage and set its dependencies.
+
+```dvc
+$ dvc stage add -n train \
+                -p model.conv_units \
+                -p train.epochs \
+                -d data/images \
+                -m metrics.json \
+                python3 src/train.py
```

Comment:
We should encourage users to update `dvc.yaml` instead of `stage add`. The command may seem useful for simple cases, but it may prevent users from learning our DSL which is what is driving all these pipelines interaction.

Related: #2578

```yaml
train:
  cmd: python3 src/train.py
  deps:
  - data/images
  params:
  - model.conv_units
  - train.epochs
  metrics:
  - metrics.json

Reply:
I think the goal is to revise once we implement https://github.com/iterative/dvc/issues/6446 to make this simpler. @skshetry we should also consider as part of `dvc exp init` and possible as a broader topic how to provide templates or other starting points for `dvc.yaml`.

Reply:
hmm, I am not sure if the `dvc exp init` is going to be any easier than the `dvc.yaml` for other scenarios. We should not try to keep them away from dvc.yaml.

Reply:
Editing `dvc.yaml` requires a node in the narrative towards pipelines, which we may not return to experiments easily. We are trying to come up with the simplest. The pipelines will have its own tutorial once we complete these. @skshetry

Reply:
I think since we don't include pipelines, and we could start this doc like:

`git clone project`
`pip install req`
`dvc pull`
`dvc exp run`

do we need to explain anything at all here?

or at least if we want to keep doin this, I would instead create an expandable section:

"Explaining structure of the project" where we explain `dvc.yaml`, put a reference to the pipelines/dvc stage add, etc.